Microsoft.Toolkit.Uwp.SampleApp/Models/Sample.cs

```diff
@@ -457,7 +457,7 @@ public async Task PreparePropertyDescriptorAsync()
                     XamlCode = await codeStream.ReadTextAsync(Encoding.UTF8);
 
                     // Look for @[] values and generate associated properties
-                    var regularExpression = new Regex(@"@\[(?<name>.+?)(:(?<type>.+?):(?<value>.+?)(:(?<parameters>.+?))?(:(?<options>.*))*)?\]@?");
+                    var regularExpression = new Regex("(?<=\\\")@\\[(?<name>.+?)(:(?<type>.+?):(?<value>.+?)(:(?<parameters>.+?))?(:(?<options>.*))*)?\\]@?(?=\\\")");
```

Comment:
@dotMorten did you remove the verbatim (@) string to reduce confusion with the starting (@)? The only issue then is it makes it harder to copy out this RegEx if we need to modify it again later and test with other RegEx helper tools.